docs/source/formulations_example.rst

```diff
@@ -0,0 +1,648 @@
+    formulation = FormulationDescriptor("mixed and blended margarita")
+    data_source = GemTableDataSource(table_id=table.uid, table_version=table.version, formulation_descriptor=formulation)
+
+The first component of the graphical model is a :class:`~citrine.informatics.predictors.simple_mixture_predictor.SimpleMixturePredictor`, which flattens the input formulation--it repeatedly replaces components with their ingredients until only the basic ingredients remain.
+This flattening efficiently teaches the predictor about the relationship between ingredients.
+In this case, it learns exactly how "simple syrup A" and "simple syrup B" are similar to each other.
+Although the homogeneous representation is not entirely appropriate for all formulations problems, it is usually an excellent approximation,
+especially when coupled with flexible machine learning models that can learn subtle relationships within the data.
+
+.. code-block:: python
+
+    from citrine.informatics.predictors import SimpleMixturePredictor
+
+    flat_formulation = FormulationDescriptor("homogenized margarita")
+    simple_mixture_predictor = SimpleMixturePredictor(
+        name="Simple margarita mixture",
+        description="Flatten a mixture of mixtures into leaf ingredients",
+        input_descriptor=formulation,  # this is the formulation descriptor from the table
+        output_descriptor=flat_formulation  # this is a new descriptor to represent the flattened formulation
+    )
+
+Using the flattened formulation as an input, we create several featurizers to compute features that will be inputs to the machine learning model.
+The featurizer predictors are :class:`~citrine.informatics.predictors.ingredient_fractions_predictor.IngredientFractionsPredictor`,
+:class:`~citrine.informatics.predictors.label_fractions_predictor.LabelFractionsPredictor`, and :class:`~citrine.informatics.predictors.mean_property_predictor.MeanPropertyPredictor`.
+We create one predictor each for ingredient and label fractions, and two mean property predictors--
+one that computes the mean price over all ingredients (this will be used to constraint the price of new margarita recipes) and one that computes the mean sucrose content of just the sweeteners.
+
+.. code-block:: python
+
+    from citrine.informatics.descriptors import RealDescriptor
+    from citrine.informatics.predictors import LabelFractionsPredictor, IngredientFractionsPredictor, MeanPropertyDescriptor
+
+    label_fractions_predictor = LabelFractionsPredictor(
+        name="Label fractions",
+        description="Total quantity that is from one of the component types",
+        input_descriptor=flat_formulation,
+        labels={"acid", "alcohol", "sweetener"},
+    )
+
+    ingredient_fractions_predictor = IngredientFractionsPredictor(
+        name="Ingredient Fractions",
+        description="Compute the fraction of each ingredient",
+        input_descriptor=flat_formulation,
+        ingredients=atomic_ingredients,
+    )
```

Comment:
MAYBE - Should we say that this is just for demonstration purposes, and that in many cases using both these predictors can result in highly correlated inputs? I typically try to use either one or the other.

SHOULD - Even though this isn't full documentation, I **strongly** suggest we explain that the `atomic_ingredients` variable is a list of Material UIDs. This is often very confusing to users. They often try a list of Ingredient Names or a list of Material Names.